### ISSUE TITLE
Make "Log to console" stick

### ISSUE THREAD

**Issue by solskogen** [BlitterStudio/amiberry#1332]
As of now, there's no way to save the option "Log to console"

**Comment by solskogen** [comment]
It doesn't make that much sense on WinUAE, but I think it might do in Amiberry (although not very much needed)
and it doesn't make much sense to NOT save that option, but save others.

**Comment by midwan** [comment]
The option can be triggered from both the console (`--log`) and the GUI (Paths panel). I don't see any scenario where this would be useful to have in a config file, considering the expected use for it (temporary troubleshooting).

Can you explain what you have in mind, that would require it?

**Comment by solskogen** [comment]
It's not about what I need, but keeping it consistent.  The user would expect it to stick when other options do.

**Comment by giantclambake** [comment]
/2cents ~ initially, I believed the reason this option was in the Paths panel, was because there was an inference that the `Enable logging` option had to be enabled first, for it to work...(from #1310 )...

> From the GUI, in the Paths panel, you will find a new checkbox (Log to console). Enable this, and the output will also go to the console

...the word 'also' being the culprit ;) Now that I check it, the `Log to console` option is actually standalone, and doesn't require logging to be enabled to work correctly.

That being the case, and taking a step back to play a game of 'one of these things is not like the others' when looking at the Paths panel, the odd man out is the `Log to console` option --- it has nothing to do with any file path =)

Given the niche usage case the folks over at TASVideo requested this option for, (and likewise they haven't come back to ask if this option could be persistent), and thinking how 'typical' amiberry users would 'normally' be using the logfile creation feature, I would wonder how many of them might enable log to console....just because that option is there =)

For mine, I would argue the `Log to console` option has very little to do wrt being a logging option, and certainly nothing to do with Paths, and given the motivation that led to it's implementation, what it really is... is ' a specific emulator option that doesn't fit in any other panel'... and that's why it should be in the Miscellaneous panel instead, because if anything I'd classify it as being a 'developer option' as it were ..which is really what it is ;) From there, it might be saved to a config.uae as well...

...I'll hedge a bet the folks at TASVideo are on the cmdline ;)

**Comment by midwan** [comment]
In the original implementation, you _had_ to enable logging in order for this to work. This was changed later, as I found it a bit better this way, but I forgot to update the help text. :)

I also suspect that most people that _want_ to use it, might do so from the console. However, following the general rule to keep the GUI similar to that of WinUAE, I've placed the checkbox there to correspond with the `Log window` option of WinUAE:

![image](https://github.com/BlitterStudio/amiberry/assets/3877365/fd5933ef-3d4b-498c-be34-dce408188cc8)

That option does not save anywhere in WinUAE either. :)
